### ISSUE TITLE
Dynamic Field Corr. option causes ICE crash w/ MB

### ISSUE THREAD

**Issue by mscohen** [CMRR-C2P/MB#124]
The physio recording output is an invaluable feature for the current CMRR releases. However, as best I can tell it will not be present in the Siemens SMS product. Are you able to share the relevant source file changes that were made to enable this? I would like to incorporate it into other current sequences, as well as to see it in the multi band sequences going forward.

Thanks.

**Comment by eauerbach** [comment]
Well, my physio logging implementation was inspired by a Siemens WiP that was originally produced for VD13x, so I think if there is going to be a version of the product sequence with logging added it would appropriately be an updated version of that WiP. Maybe that already exists, I don't know.

**Comment by mscohen** [comment]
Oops. I spoke to soon. This time the sequence crashed with a different error:

"Too high gradient pulses for too long time (Duty Cycle)
Please check/modify the sequence parameters, start the sequence again, reboot "Meas&Recon" or switch on/off the scanner."

I will create the save log for this as well. Perhaps I am pushing the gradients too hard. If you have a dcm file of the HCP 64 channel direction diffusion sequence to share, perhaps I can capture the correct parameters.

Thanks sincerely for all of your support.

Mark S. Cohen
Staglin Center for Cognitive Neuroscience
Room C9-420
Semel Institute for Neuroscience & Human Behavior
760 Westwood Plaza, Los Angeles, CA 90095

mscohen@g.ucla.edu
310-980-7453
http://tinyurl.com/mscohen
http://www.ucla-cir.org

**Comment by mscohen** [comment]
Hi there,

Mark Cohen (mscohen@ucla.edu) invited you to edit the folder " CMRR " on Dropbox.

Mark said:
"Good evening:

I have loaded 3 savelogs into dropbox after the crashes I noted in my email.

The dropbox link to the shared files is: https://www.dropbox.com/l/sh/Kl2UH58AUm4By8TPMMxWCx?text=1

The first followed a scan with the Resolution->Filter->Dynamic Field Corr. turned on.

After turning off that option, I ran another trial which came close to completion, but stopped with a gradient duty-cycle error. I have checked with service about this.

I reduced the maximum b value from 1500 to 1400, and ran again. The final savelog is from this scan and reports an RF safety watchdog error. I checked with service about this, as well. As this problem occurred yesterday in an unrelated scan, it may or may not be relevant to the CMRR questions.

Thanks for your willingness to help."

Go to folder[1]

Enjoy!
The Dropbox team

[1]: https://www.dropbox.com/l/scl/F6AHQrCfHOnPSENtwzvSPp?text=1
